com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -2976,7 +2982,7 @@ void Callback(CommandBuffer c, HDCamera cam)
             RenderCustomPass(renderContext, cmd, hdCamera, customPassCullingResults, CustomPassInjectionPoint.BeforePostProcess, aovRequest, aovCustomPassBuffers);
 
             if (aovRequest.isValid)
-                aovRequest.PushCameraTexture(cmd, AOVBuffers.Color, hdCamera, m_CameraColorBuffer, aovBuffers);
+                aovRequest.PushCameraTexture(cmd, AOVBuffers.Color, hdCamera, GetDebugViewTargetBuffer(), aovBuffers);
```

Comment:
Could probably be removed?

Reply:
I am not sure to understand what the change do here?